helm-chart/templates/ingress.yaml

```diff
@@ -24,7 +24,14 @@ spec:
     - host: {{ index .Values.ingress.hosts 1 | quote }}
       http:
         paths:
-            - path: /.+/(graphql|sitemap\.xml|docs)
+            - path: /apps
+              pathType: Prefix
+              backend:
+                service:
+                  name: {{ $fullName }}
+                  port:
+                    number: 8080
+            - path: /[^/]+/(api|index|tables|molgenis-components|schema|updownload|reports|graphql-playground|tailwind-components|ui|docs|graphql|sitemap\.xml|theme\.css)
```

Comment:
this list should include all the apps now plus a few special endpoints on schema level such as api,  index, graphql, sitemap and theme.csss